conda_forge_tick/make_graph.py

```diff
@@ -224,10 +224,10 @@ def update_graph_pr_status(gx: nx.DiGraph) -> nx.DiGraph:
     return gx
 
 
-def close_labels(gx: nx.DiGraph) -> nx.DiGraph:
+def close_labels(dry_run, gx: nx.DiGraph) -> nx.DiGraph:
```

Comment:
Should the args be reversed with `dry_run` as a kwarg defaulted to False so it doesn't break the existing api? Also `update_graph_pr_status`?